lib/kie/service.rb

```diff
@@ -0,0 +1,27 @@
+require 'kie_client'
+module Kie
+  class Service
+    def self.call(klass, options)
+      setup(options)
+      yield init(klass)
+    rescue KieClient::ApiError => err
+      Rails.logger.error("KieClient::ApiError #{err.message} ")
+      raise Exceptions::KieError, err.message
+    end
+
+    private_class_method def self.setup(options)
+      KieClient.configure do |config|
+        config.host     = options['host'] || 'localhost'
+        config.scheme   = options['host'].try(:scheme) || 'http'
+        config.username = options['username']
+        config.password = options['password']
```

Comment:
This is a temporary solution. We will use manageiq-api-common to manage passwords in a follow-up PR.